cmd/main.go

```diff
@@ -54,7 +54,9 @@ var (
 
 	// Feature lock release specific parameters, only take effect when feature-lock-release is set to true.
 	featureLockRelease = flag.Bool("feature-lock-release", false, "if set to true, the node driver will support Filestore lock release.")
-
+	// featureLockRelease must be set as true when featureLockReleaseV2 is true. V2 implementation will override part of v1 implementation when true.
+	featureLockReleaseV2  = flag.Bool("feature-lock-release-v2", false, "if set to true, the node driver will not support v1 Filestore lock release.")
```

Comment:
Since we’re the only ones familiar with the differences between v1 and v2, should we consider naming it `standalone-lock-release-controller` to make it more intuitive for others?